### PR TITLE
Create cropper documentation

### PR DESCRIPTION
**Pull request by fakela**

### CODE REVIEW

**Comment by fakela** [comment]
@fharper I made some last-minute changes.

File: apis-overview/endpoints/cropper.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+---
+
+> 📘**Info**
+>
+> This feature is available as part of the [Mindee Enterprise plan](https://mindee.com/pricing).
+
+## What is Cropper?
+Cropper enables users to retrieve the coordinates of cropped images from a document through our APIs.
+
+Mindee API prediction endpoint works with one document per request (image or PDF file). The cropper results are at the page level and each page can detect several _documents_. Most of the time, taking the first element of the cropper output is sufficient for any use case.
```
Comment (fharper) [inline apis-overview/endpoints/cropper.md:14]: I don't understand "The cropper results are at the page level and each page can detect several _documents_."

Comment (fharper) [inline apis-overview/endpoints/cropper.md:14]: for any use case = for most use case

File: apis-overview/endpoints/cropper.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+## How it works
```
Comment (fharper) [inline apis-overview/endpoints/cropper.md:16]: Don't we use capital letters in header?

File: apis-overview/endpoints/cropper.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+The results shows the polygon vertices at the page level, for each possible cropped document within the image:
+- **bounding_box**: straight rectangle (always inside canvas)
+- **rectangle**: rectangle that may be oriented (can go beyond the canvas, ie. vertices coordinates < 0 and > 1)
+- **quadrangle**: free polygon with 4 vertices (always inside canvas)
+- **polygon**: free polygon with up to 30 vertices (always inside canvas)
+
+The Vertices format are a list of (x, y) relative coordinates, always in clockwise order. Starting vertex may not be fixed (depends if this is a rectangle or a free polygon).
+Example: `[[0.143, 0.002], [0.727, 0.002], [0.727, 0.996], [0.143, 0.996]]`.
+
+### Example with receipts V3
```
Comment (fharper) [inline apis-overview/endpoints/cropper.md:36]: Don't we use capital letters in header?

File: apis-overview/endpoints/cropper.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+>
+> This feature is available as part of the [Mindee Enterprise plan](https://mindee.com/pricing).
+
+## What is Cropper?
+Cropper enables users to retrieve the coordinates of cropped images from a document through our APIs.
+
+Mindee API prediction endpoint works with one document per request (image or PDF file). The cropper results are at the page level and each page can detect several _documents_. Most of the time, taking the first element of the cropper output is sufficient for any use case.
+
+## How it works
+The cropper feature can be used on any API from Mindee API V1 on the prediction route:
```
Comment (fharper) [inline apis-overview/endpoints/cropper.md:17]: I think we can remove the "from Mindee API V1"

File: apis-overview/endpoints/cropper.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+```http
+https://api.mindee.net/v1/products/<account_name>/<api_name>/<api_version>/predict?cropper=true
+```
+ `?cropper=true` which is the additional parameter
+
+The cropper results are stored as follows
```
Comment (fharper) [inline apis-overview/endpoints/cropper.md:24]: are returned as follows in the JSON object

or something in those lines

File: apis-overview/endpoints/cropper.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+- **bounding_box**: straight rectangle (always inside canvas)
+- **rectangle**: rectangle that may be oriented (can go beyond the canvas, ie. vertices coordinates < 0 and > 1)
+- **quadrangle**: free polygon with 4 vertices (always inside canvas)
+- **polygon**: free polygon with up to 30 vertices (always inside canvas)
+
+The Vertices format are a list of (x, y) relative coordinates, always in clockwise order. Starting vertex may not be fixed (depends if this is a rectangle or a free polygon).
+Example: `[[0.143, 0.002], [0.727, 0.002], [0.727, 0.996], [0.143, 0.996]]`.
+
+### Example with receipts V3
+In our POST request, the cropper additional param `?cropper=true` is added to the receipts V3 API from Mindee API V1.
```
Comment (fharper) [inline apis-overview/endpoints/cropper.md:37]: You should add the document you use to make this example.

File: apis-overview/endpoints/cropper.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+               }
+             ]
+           }
+         }
+       }
+     ]
+   }
+ }
+}
+```
```
Comment (fharper) [inline apis-overview/endpoints/cropper.md:113]: We may want to add the initial example document with the polygon vertices highlighted, so it's more visual.

**Review by fharper — CHANGES_REQUESTED** [review]

File: apis-overview/endpoints/cropper.md [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+- **quadrangle**: free polygon with 4 vertices (always inside canvas)
+- **polygon**: free polygon with up to 30 vertices (always inside canvas)
+
+The Vertices format are a list of (x, y) relative coordinates, always in clockwise order. Starting vertex may not be fixed (depends if this is a rectangle or a free polygon).
+Example: `[[0.143, 0.002], [0.727, 0.002], [0.727, 0.996], [0.143, 0.996]]`.
+
+### Example with receipts V3
+In our POST request, the cropper additional param `?cropper=true` is added to the receipts V3 API from Mindee API V1.
+
+POST `https://api.mindee.net/v1/products/mindee/expense_receipts/v3/predict?cropper=true`
```
[the remark this answers is not in the document]
Comment (fakela) [reply apis-overview/endpoints/cropper.md:39]: I can try for curl but we will have to test its accuracy

File: apis-overview/endpoints/cropper.md (hunk 7 above)
Comment (fakela) [reply apis-overview/endpoints/cropper.md:143]: I don’t understand this one

Comment (fharper) [reply apis-overview/endpoints/cropper.md:144]: I wanted to show something like the document with the polygons of the cropper shown on it, so people see what they get, but it's not urgent, and I'll do it as I really want to finally use our Computer Vision SDK :)

**Review by fharper — APPROVED** [review]
LGTM

**Pull request merged**